middlewares/authorizeBot.js

```diff
@@ -0,0 +1,20 @@
+const botVerifcation = require("../services/botVerificationService");
+const { CLOUDFLARE_WORKER } = require("../constants/bot");
+
+module.exports = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization.split(" ")[1];
+    const data = botVerifcation.verifyToken(token);
```

Comment:
why arent we checking the expiry of JWT here?

Reply:
So `jwt.verify` doesn't verify the expired token and ultimately throws an error. Therefore, the control goes to `catch` block and there we send the response of `Invalid Request`.
Do we want the user of this API to know that the token is expired?
My POV is we can do it for the developers to get better errors and debug the code faster if incase any bug arises.

Reply:
ok
Would be easy if you could show a demo video.

Reply:
[Screencast from 2023-01-31 19-55-13.webm](https://user-images.githubusercontent.com/78433013/215787664-d23ab69e-9fa8-423b-80c9-5de3e767a68a.webm)

Reply:
but in our cfw we are setting expiry in jwt header
what about that scenario?